zstackctl/zstackctl/ctl.py

```
@@ -3201,6 +3201,15 @@ def check_simulator():
         prepare_bean_ref_context_xml()
 
         linux.sync_file(ctl.properties_file_path)
+
+        # clean up unexpected __pycache__ under ansible/files/ before MN starts
+        ansible_files_dir = os.path.join(ctl.zstack_home, "../../ansible/files")
+        if os.path.isdir(ansible_files_dir):
+            for root, dirs, _ in os.walk(ansible_files_dir):
+                for d in dirs:
+                    if d == '__pycache__':
+                        rmtree(os.path.join(root, d), ignore_errors=True)
```

Ruff (0.15.6)

[error] `os` may be undefined, or defined from star imports

(F405)